troubleshooting/app-does-not-connect-to-remote-point.md

```diff
@@ -0,0 +1,47 @@
+---
+title: App Cannot Connect to a Remote Service via HTTPS When Running on Device
+description: App cannot connect to a web service via HTTPS when deployed on a device and produces an unspecified error.
+page_title: App Cannot Connect to Remote Service via HTPS When Running on Device
```

Comment:
We don't need to change it everywhere as clients' searches will vary. Both can be used.